src/content/guides/getting-started.md

```diff
@@ -13,36 +13,36 @@ contributors:
   - TheDutchCoder
   - sudarsangp
   - Vanguard90
+  - chrischinchilla
 ---
 
-As you may already know, webpack is used to compile JavaScript modules. Once [installed](/guides/installation), you can interface with webpack either from its [CLI](/api/cli) or [API](/api/node). If you're still new to webpack, please read through the [core concepts](/concepts) and [this comparison](/comparison) to learn why you might use it over the other tools that are out in the community.
-
+Webpack is used to compile JavaScript modules. Once [installed](/guides/installation), you can interface with webpack either from its [CLI](/api/cli) or [API](/api/node). If you're still new to webpack, read through the [core concepts](/concepts) and [this comparison](/comparison) to learn why to use it over the other tools available the community.
 
 ## Basic Setup
 
-First let's create a directory, initialize npm, and [install webpack locally](/guides/installation#local-installation):
+Create a directory, initialize npm, and [install webpack locally](/guides/installation#local-installation):
 
-``` bash
+```bash
 mkdir webpack-demo && cd webpack-demo
 npm init -y
 npm install --save-dev webpack
 ```
 
-Now we'll create the following directory structure and contents:
+Create the following directory structure and contents:
 
-__project__
+**project**
```

Comment:
We use double underscores everywhere. It makes no difference, but no need to change it.

Reply:
Sorry, that's my Markdown linter there, always changes the formatting used :(